website/content/docs/connect/cluster-peering/create-manage-peering.mdx

```diff
@@ -0,0 +1,137 @@
+
+When you connect server agents through cluster peering, they will peer their default partitions. To establish peerings for other partitions through server agents, you must specify the partitions you want to peer using the `Partition` field of the request body.
+
+## Connect service endpoints
+
+After you establish a connection between the clusters, you need to create a configuration entry that defines the services that are available for other clusters. Consul uses this configuration entry to replicate service definitions and support service mesh connections across clusters.
+
+First, create a configuration entry and specify the `Kind` as `“exported-services”`.
+
+<CodeBlockConfig filename="peering-config.hcl" highlight="1" hideClipboard>
+
+```shell-session
+Kind = "exported-services"
+Partition = "partition-name"
+
+Services = [
+  {
+    ## The name and namespace of the service to export.
+    Name      = "service-name"
+    Namespace = "default"
+
+    ## The list of peer clusters to export the service to.
+    Consumers = [
+      {
+        ## The peer name to reference in config is the one set
+        ## during the peering process.
+        Peer = "cluster-02"
+      }
+    ]
+```
+
+</CodeBlockConfig>
+
+Then, add the configuration entry to your cluster.
+
+```shell-session
+$ consul config write peering-config.hcl
+```
+
+Before you proceed, wait for the clusters to sync and make services available to their peers
+
+## Authorize connections from peers
+
+Before you can call services from peered clusters, you must set service intentions that authorize those clusters to use specific services. Consul prevents services from being exported to unauthorized clusters.
+
+First, create a configuration entry and specify the `Kind` as `“service-intentions”`. Declare the service on "cluster-02" that can access the service in "cluster-01." The following example sets service intentions so that "backend-service" can access "frontend-service."
+
+<CodeBlockConfig filename="peering-intentions.hcl" highlight="1" hideClipboard>
+
+```shell-session
+Kind      = "service-intentions"
+Name      = "frontend-service"
+Partition = "partition-name"
+
+Sources = [
+  {
+    Name   = "backend-service" 
+    Peer   = "cluster-02"
+    Action = "allow"
+  }
+]
+```
+
+</CodeBlockConfig>
+
+**Tip:** If the peer’s name is not specified in `Peer`, then Consul assumes that the service is in the local cluster.
+
+Then, add the configuration entry to your cluster.
+
+```shell-session
+$ consul config write peering-intentions.hcl
+```
+
+## Check peered cluster status
+
+To confirm that you peered your clusters, you can [query the `/health/service` endpoint](https://www.consul.io/api-docs/health) of one cluster from the other cluster. For example, in "cluster-02," query the endpoint and add the `peer=cluster-01` parameter to the end of the URL.
+
+```shell-session
+$ curl \
+    http://127.0.0.1:8500/v1/health/service/service-name?peer=cluster-01
```

Comment:
```suggestion
    "http://127.0.0.1:8500/v1/health/service/service-name?peer=cluster-01"
```
I recommend quoting this URL to avoid issues with Unix shells trying to interpret the `?` is a glob character.